CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ Added
 Changed
 ~~~~~~~~
 * Introduce github actions for tests instead of TravisCI
+* Replace pygeodesy with pyproj for speed-up
```

Comment:
leaving this comment to not forget to run a comparison for a race and aat. would like to check out the results and performance increase

Reply:
tried this out through the complete loop of pysoar. indeed great performance increase and no differences in output!